src/app/cube/browse/browse.component.html

```diff
@@ -2,8 +2,8 @@
   <div class="browse-columns">
     <div class="sidebar-wrapper" [ngClass]="{'active': filtersDownMobile}">
       <div class="column-title">
-        Filters
-        <div class="filters-clear-all" (click)="clearAllFilters()">Clear all filters</div>
+        {{ copy.FILTERS }}
```

Comment:
Is this descriptive enough, or should it be something along the lines of `FILTERS_TITLE`